### PR TITLE
TA-665:Align the display of tasks' due date

### PR DESCRIPTION
**Pull request by Jihed525**

### CODE REVIEW

File: task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less [hunk 1]
```diff
@@ -1504,7 +1504,7 @@ li[data-hiddenlabel="false"] {
 }
 
 .table-project .column-time {
-    width: 27px;
+    width: auto;
```
Comment (thomasdelhomenie) [inline task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: This reduces the width of the center column, which leads to unused space and truncated long task titles, as you can see in this screenshot:

We should find a way to fix this issue without reducing the width of the center column.

Comment (Jihed525) [reply task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: Well noted, will definitely take that into consideration next time.

File: task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less [hunk 2]
```diff
@@ -1504,7 +1504,7 @@ li[data-hiddenlabel="false"] {
 }
 
 .table-project .column-time {
-    width: 27px;
```
Comment (boubaker) [inline task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: Tha changes to make it flexible by giving the title the max width:

```css
.table-project .row-item {
    display: flex;
}
.table-project .row-item .column-title {
    flex: 1;
}
.table-project .column-time {
    width: auto;
}
```

Comment (Jihed525) [reply task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: Will get them in place ASAP, Thank you !

File: task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less [hunk 3]
```diff
@@ -288,6 +288,9 @@ li[data-hiddenlabel="false"] {
     .tooltip {
         white-space: nowrap;
     }
+    .table-project .row-item {
+    min-height: 38px;
```
Comment (boubaker) [inline task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: Please pay attention to code format

Comment (Jihed525) [reply task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: Sorry for this recurrent mistake, code format updated accordingly.

**Pull request merged**